src/Serilog/Parsing/MessageTemplateParser.cs

```diff
@@ -161,47 +163,44 @@ static bool TrySplitTagContent(string tagContent, out string propertyNameAndDest
                 propertyNameAndDestructuring = tagContent;
                 format = null;
                 alignment = null;
+                return true;
```

Comment:
i did some early returns to simplify the branching logic here. let me know if u prefer it back the previous way

Reply:
LGTM :+1: